screenshot-testing/src/main/java/com/stripe/android/screenshottesting/PaparazziRule.kt

```diff
@@ -89,6 +89,7 @@ class PaparazziRule(
 
     private fun createPaparazzi(deviceConfig: DeviceConfig): Paparazzi {
         return Paparazzi(
+            maxPercentDifference = 0.001,
```

Comment:
out of curiosity: where was the max percent difference set previously? I would've expected to see it be updated instead of just added

Reply:
The default happened in the constructor, it was set at `.1`.

Reply:
ah makes sense, thanks!